src/common.tsx

```diff
@@ -0,0 +1,22 @@
+import {ApiPromise} from "@polkadot/api";
+
+interface AccountData {
+  [free: string]: any;
+}
+
+// Query account balance for bridged assets (ETH and ERC20)
+async function queryPolkaEthBalance(
+  api: ApiPromise,
+  assetId: string,
```

Comment:
> @jakai7 Looking over this, I realise we need to generate the Typescript types for our parachain so we can initialise the API correctly. You shouldn't have to define the `AccountData` type manually.
> 
Okay. Makes sense.

> Can you give me a day or 2 to implement this?
Sure, no problem, I am going to close this PR anyways because it diverted from the master branch. I will work on this when we generate the types for our parachain.

Reply:
> assetId and accountId should be type `any`. They are not strings.

I was unsure about this. Noted, thank you.